website/docs/data-modeling-concepts.mdx

```diff
@@ -0,0 +1,38 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Data Modeling Concepts
+### Models
+
+Models are [Ceramic streams](https://developers.ceramic.network/learn/advanced/overview/#streams) storing metadata about [documents](https://composedb.js.org/docs/0.3.x/guides/concepts-overview#documents). This metadata includes their data structure/schema, [validation rules](https://composedb.js.org/docs/0.3.x/guides/concepts-overview#validation) and [relations](https://composedb.js.org/docs/0.3.x/guides/concepts-overview#relations), as well as [discovery information](https://composedb.js.org/docs/0.3.x/guides/using-composites/discovery).
```

Comment:
As I tried to explain in our call, you should never have links starting from `https://composedb.js.org`, always relative links to the target pages.
Especially here, you're pointing to v0.3 docs while working on a newer version.

Reply:
I understand that we want relative links - and virtually all of the ones I added are - but we're updating this preview doc in multiple steps, so the docs content is up before the guides will be in 2 weeks.

Reply:
This is not resolved - please don't put any link starting from `https://composedb.js.org`